filters/extract_area.go

```diff
@@ -0,0 +1,111 @@
+package filters
+
+import (
+	"github.com/zalando/skipper/filters"
+	"gopkg.in/h2non/bimg.v1"
+	"strconv"
+	"strings"
+)
+
+const (
+	ExtractArea    = "transformFromQueryParams"
```

Comment:
This is the name of the filter. can you make the 2 names consistent? either extract area or transform from query param